exercises/practice/high-scores/.docs/instructions.append.md

```diff
@@ -0,0 +1,7 @@
+# ignore
+
+## Further reading
+
+You can read more about OO programming in Lua [in this article][oop].
+
+[oop]: https://exercism.org/tracks/lua/exercises/high-scores/articles/oop
```

Comment:
From https://exercism.org/tracks/lua/exercises/high-scores
when I click the `in this article` link I end up back at
https://exercism.org/tracks/lua/exercises/high-scores

Reply:
That's strange... I'm not seeing that in my browser

Reply:
Chrome and Firefox have similar behaviour for me:

When not logged in to Exercism, the link goes to glennj's article "Object Oriented Programming in Lua"

But when logged in to Exercism, I instead end up at the same 'High Scores' page I was on before.  This happens even if I paste https://exercism.org/tracks/lua/exercises/high-scores/articles/oop in the address bar.  Perhaps it is because I have not completed the exercise?

Reply:
Maybe because it's a "dig deeper" article. I'll post it elsewhere and change the link